modules/cpmso-yaml-sample-cr.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control-plane-machine-management.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-yaml-sample-cr_{context}"]
+= Sample YAML for a control plane machine set custom resource
+
+The base of the `ControlPlaneMachineSet` CR is structured the same way for all platforms.
+
+.Sample `ControlPlaneMachineSet` CR YAML file
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1
+kind: ControlPlaneMachineSet
+metadata:
+  name: cluster <1>
+  namespace: openshift-machine-api
+spec:
+  replicas: 3 <2>
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-machine-role: master
+      machine.openshift.io/cluster-api-machine-type: master
```

Comment:
need to add ` state: Inactive`

Reply:
I think it will be `Active` for a new AWS 4.12 cluster, right? I want it to reflect that case, and explain in the notes about both values.